apps/charterafrica/src/payload/globals/Footer.js

```diff
@@ -0,0 +1,99 @@
+const Footer = {
+  slug: "Footer",
+  label: {
+    en: "Footer",
+    fr: "Bas de page",
+    pt: "Rodapé",
+  },
+  fields: [
+    {
+      name: "siteDescription",
+      defaultValue:
+        "This website was created and maintained with the financial support of the European Union. Its contents are the sole responsibility of the European Partnership for Democracy, Africtivistes, Code for Africa, ECPDM, and Goree Institute and do not necessarily reflect the views of the European Union.",
+      label: {
+        en: "Site Description",
+        pt: "Descrição do Site",
+      },
+      type: "textarea",
+      localized: true,
+      required: true,
+    },
+    {
+      name: "projectDescription",
+      defaultValue: "Website designed and built by Code for Africa",
+      label: {
+        en: "Project Description",
+        pt: "Descrição do Projeto",
+      },
+      type: "text",
+      localized: true,
+      required: true,
+    },
+    {
+      name: "contactEmail",
+      defaultValue: "info@charter.africa",
+      label: {
+        en: "Contact Email",
+        pt: "Email de contato",
+      },
+      type: "text",
```

Comment:
Use type group with name `contact` with a child field named `email` (with type `email` as well) so that we end up with a nice contact object.